java/src/main/java/ai/rapids/cudf/HostColumnVector.java

```diff
@@ -796,6 +805,15 @@ public static HostColumnVector fromBoxedUnsignedShorts(Short... values) {
     return build(DType.UINT16, values.length, (b) -> b.appendBoxed(values));
   }
 
+  /**
+   * Create a new vector from the given values.  This API supports inline nulls,
+   * but is much slower than using a regular array and should really only be used
+   * for tests.
+   */
+  public static HostColumnVector fromDurationDaysInBoxedInts(Integer... values) {
```

Comment:
Recommend renaming this `durationDaysFromBoxedInts` to be consistent with `ColumnVector` and the existing `timestampDaysFromBoxedInts` method in this class.